core/src/main/kotlin/org/jetbrains/research/testspark/core/generation/llm/LLMWithFeedbackCycle.kt

```diff
@@ -80,6 +80,9 @@ class LLMWithFeedbackCycle(
         var executionResult = FeedbackCycleExecutionResult.OK
         val compilableTestCases: MutableSet<TestCaseGeneratedByLLM> = mutableSetOf()
 
+        // collect imports from all responses
+        val imports: MutableList<String> = mutableListOf()
```

Comment:
**nitpick:** We can make it `MutableSet<String>` so that duplicates are removed as we go. The current implementation is also correct, but a set, in my opinion, fits better ideologically.